ably.d.ts

```
@@ -2990,3 +2995,6 @@ export declare class ErrorInfo extends Error {
    */
   constructor(message: string, code: number, statusCode: number, cause?: string | Error | ErrorInfo);
 }
+
+type NonFunctionKeyNames<A> = { [P in keyof A]: A[P] extends Function ? never : P }[keyof A];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)